package.json

```diff
@@ -23,6 +23,7 @@
     "apollo-link-context": "1.0.17",
     "apollo-link-error": "1.1.10",
     "apollo-link-http": "1.5.14",
+    "blockstack": "^19.2.0-beta.1",
```

Comment:
Why use the beta?

Reply:
Upgraded to 19.2.1 in https://github.com/opencollective/opencollective-frontend/pull/1980